app/workers/company_video_worker.rb

```diff
@@ -0,0 +1,35 @@
+class CompanyVideoWorker
+  include Sidekiq::Worker
+
+  def perform(company_id)
+    company = Company.find(company_id)
+    if company.youtube?
+      channel = get_channel(company.youtube)
+      channel_videos = get_videos(channel)
```

Comment:
it's more easier to make it
```
channel_videos = channel.videos
```
instead of creating separate method

Reply:
I need to mock it in tests